src/packet.rs

```diff
@@ -19,6 +20,9 @@ pub enum RtpParseError {
     /// The padding byte does not contain a valid value.
     #[error("Padding contains invalid value {}", .0)]
     PaddingInvalid(u8),
+    /// This is actually an RTCP packet.
+    #[error("RTCP packet of type {}", .0)]
```

Comment:
Could implement `RtpParseError::is_rtcp_packet()` so users don't have to `if matches!(e, rtp_types::RtpParseError::RtcpPacket(_))`.